### ISSUE TITLE
Min feedback

### ISSUE THREAD

**Issue by TheWonkyTestube** [minbrowser/min#1610]
I can't speak highly enough of this project, the browser is smooth, easy to use, and of course very minimal. My only issue is that the browser has no homepage system that I can find, which for a browser this minimal, I understand. However, if a custom homepage is off the cards, maybe the background of the New Tab page could match the theme for the rest of the browser? It is pretty glaring on a computer with a dark UI setup, and with the browser itself on Dark mode!

Again, other than this very minor gripe, I have no other criticisms, a superb project.

**Comment by Syndamia** [comment]
Around half a year ago, me and @Milo123459 woked on a home page, but at the end we dropped it, since there were more important things to do and because that kinda conflicted with Palmer's vision (if I remember correctly).

I'll be leaving this issue open for now, but could you please change the title? "Min feedback" is a bit too vague, changing it to something like "Custom homepage or custom New Tab page background?" will be much appreciated!

**Comment by TheWonkyTestube** [comment]
Of course! Thanks for your reply, and thanks for the project, its really
awesome!

**Comment by Milo123459** [comment]
Yea, we did work on something for it. The idea kinda died out however.

**Comment by PalmerAL** [comment]
IIRC, my thought at the time was that having a traditional home page with a search box and list of bookmarks wouldn't be very useful because the searchbar already contains that information, and is distracting when opening a new tab.

> However, if a custom homepage is off the cards, maybe the background of the New Tab page could match the theme for the rest of the browser? It is pretty glaring on a computer with a dark UI setup, and with the browser itself on Dark mode!

Agree. The history here is that this was enabled at one point, and then got disabled due to bugs in Electron: https://github.com/minbrowser/min/issues/846

But I think it should be possible to work around that issue and re-enable it.

**Comment by swang203** [comment]
I think it is a killer feature compared to chrome/brave/edge. These browsers doesn't have a nice way to set new tab background/picture by default, users have to install plug-in to achieve that. Brave only have several pictures poorly cycling all the time (pretty boring). Looking forward that min allows user to upload their picture as the tab background!
